test.rb

```diff
@@ -0,0 +1,5 @@
+if foo
+  foo.bar
```

Comment:
__[development][rubocop]__
* Use safe navigation (`&.`) instead of checking if an object exists before calling the method.

<!-- Comment by [SideCI](https://sideci.com) -->